torch/testing/_internal/common_methods_invocations.py

```diff
@@ -590,6 +590,7 @@ def method_tests():
         ('view', (S,), (S,), '1d', (False,)),
         ('view', (), (dont_convert(()),), 'scalar_to_scalar', (False,)),
         ('view', (), (1,), 'scalar_to_1d', (False,)),
+        ('ravel', (S, S, S), NO_ARGS, '', (True,)),
```

Comment:
I think the (True,) should be (False,): this boolean represents whether or not this operator is supported by JIT autodiff, which is not the same thing as autograd. Since we didn't make any changes to the JIT in this PR to support `ravel` in JIT autodiff, setting this as (True,) should cause some of the JIT tests to fail.

In general it is OK for an operator to not be supported by JIT autodiff (in fact, most operators aren't supported). JIT autodiff is a symbolic differentiation pass that is useful for the JIT when it tries to fuse things in the backwards graph.

Reply:
I see. I though autodiff is similar to autograd. Haha, I may need to also check the JIT.